tasks/7.html

```diff
@@ -12,7 +12,10 @@ <h1 class="global-heading">&nbsp;</h1>
 
         <main class="global-results">
             <style>
-                /* your styles here */
+                [src='http://www.codingdojo.com/blog/wp-content/uploads/css.jpg']:not([alt])
```

Comment:
Можно просто указать тег)
```suggestion
                img:not([alt])
```